### ISSUE TITLE
Tally still displays last status when vMix software is closed

### ISSUE THREAD

**Issue by Dirwinc** [guido-visser/vMix-M5Stick-Tally-Light#9]
Thanks for all the updates on this Guido, I really appreciate all your work.  I'm testing out the latest v1.4 version now.   

When the tally light is operating normally... and then I close vMix down.... the tally continues to display the last input status from vMix before it was closed (either Safe or Pre or Live... whatever the last status was before vMix shut down).  Shouldn't the Tally automatically sense that the vMix connection was lost, and go back into the attempting to reconnect to vMix routine?

And then when I start vMix again, the Tally light is still stuck on the last status before the shut down... it doesn't sense vMix has restarted and the inputs are changing again.

I notice that when I press button B on the tally that it fixes it and 'unfreezes' the status.  But where I have the tally mounted, I cannot do that during a live production, so I'd like to request the code to automatically sense a vMix shutdown and attempt to reconnect (without the button B press).  

Thanks!

**Comment by guido-visser** [comment]
Hi Driwin,

Thanks for the issue. Are you sure you uploaded v1.4? When vMix closes, a disconnection screen should show now. I do agree that the stick should try to reconnect automatically, however I ran into an issue where during reconnecting to vMix, the webserver which supplies the Web UI isn't being upheld, because it's trying to reconnect to vMix.

I could look at the issue again to see if both can happen at the same time. After a failed third attempt, it will then show the disconnection screen.

In v1.4 you can also let the stick reconnect using the Web UI. I think that I'll also show the version number on the main boot screen for clarification.

Would this be a good solution for you?

**Comment by Dirwinc** [comment]
Hi Guido, 

Yeah, something odd is happening on my sticks (I have two sticks).  I just reinstalled 1.4 (I swear I had it on there before, but I could be wrong), and you're right... it is displaying the "Couldn't connect to vmix" screen now when I try to reproduce it.  So no issue now (we can close this issue).

I like you're idea of putting the build version on the screen somewhere, that works for me.

I also like you're idea of a failed third connection attempt (or so) and then show the disconnection screen.  I think this will help with some of the network issues I've been having on my church network, but not on my home network.

btw, I just ordered the M5StickC NeoFlash Hat to see if I can get the add-on LED matrix to work with your code (not sure if I'll be successful though :)).  My stage talent is having trouble seeing the stick, so I'm hoping this will make it a lot brighter for them.  
https://m5stack.com/collections/m5-hat/products/m5stickc-neofalsh-hat

Thanks!
Dirwin
